build failure

ich kriege einen Build Failure für 136f923219ac9bb911af6a4ea5041461f4c1c6b4
```
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:3: error: package com.sun.tools.javac.util does not exist
import com.sun.tools.javac.util.Pair;
                               ^
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:18: error: cannot find symbol
    public Pair<InputStream, FileType> detect(InputStream stream) {
           ^
  symbol:   class Pair
  location: class Detector
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:3: error: package com.sun.tools.javac.util does not exist
import com.sun.tools.javac.util.Pair;
                               ^
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:27: error: cannot find symbol
                return new Pair(s, FileType.OrderResponse);
                           ^
  symbol:   class Pair
  location: class Detector
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:29: error: cannot find symbol
                return new Pair(s, FileType.DispatchNotification);
                           ^
  symbol:   class Pair
  location: class Detector
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:31: error: cannot find symbol
                return new Pair(s, FileType.Order);
                           ^
  symbol:   class Pair
  location: class Detector
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java:33: error: cannot find symbol
                return new Pair(s, FileType.Invoice);
                           ^
  symbol:   class Pair
  location: class Detector
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:19: error: cannot find symbol
        Pair<InputStream, Detector.FileType> result = new Detector().detect(stream);
        ^
  symbol:   class Pair
  location: class OpenTransReader
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:20: error: illegal start of type
        switch (result.snd) {
               ^
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:21: error: cannot find symbol
            case Invoice:
                 ^
  symbol:   variable Invoice
  location: class OpenTransReader
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:23: error: cannot find symbol
            case OrderResponse:
                 ^
  symbol:   variable OrderResponse
  location: class OpenTransReader
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:25: error: cannot find symbol
            case DispatchNotification:
                 ^
  symbol:   variable DispatchNotification
  location: class OpenTransReader
/home/zdavatz/software/yopenedi/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java:27: error: cannot find symbol
            case Order:
                 ^
  symbol:   variable Order
  location: class OpenTransReader
13 errors
```

--- a/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java
+++ b/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/Detector.java
@@ -1,6 +1,6 @@
 package com.ywesee.java.yopenedi.OpenTrans;
 
-import com.sun.tools.javac.util.Pair;
+import com.ywesee.java.yopenedi.converter.Pair;
 
 import java.io.IOException;
 import java.io.InputStream;

--- a/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java
+++ b/java/converter/src/main/java/com/ywesee/java/yopenedi/OpenTrans/OpenTransReader.java
@@ -1,6 +1,6 @@
 package com.ywesee.java.yopenedi.OpenTrans;
 
-import com.sun.tools.javac.util.Pair;
+import com.ywesee.java.yopenedi.converter.Pair;
 import com.ywesee.java.yopenedi.converter.Utility;
 
 import javax.xml.namespace.QName;

--- a/java/converter/src/main/java/com/ywesee/java/yopenedi/converter/Pair.java
+++ b/java/converter/src/main/java/com/ywesee/java/yopenedi/converter/Pair.java
@@ -1,0 +1,10 @@
+package com.ywesee.java.yopenedi.converter;
+
+public class Pair<A, B> {
+    public A fst;
+    public B snd;
+    public Pair(A a, B b) {
+        this.fst = a;
+        this.snd = b;
+    }
+}

